tests/find_and_replace/find_and_replace_plugin.test.ts

```diff
@@ -697,3 +718,25 @@ describe("replace", () => {
     expect(getActivePosition(model)).toBe("A1");
   });
 });
+
+test("simple search with array formula", () => {
```

Comment:
test name, we're testing replace. "replace do'nt replace spreaded results of array formula" or something like that